File: clases/09_sesion_de_problemas/exceso_de_tela/exceso_de_tela.cpp
#include <iostream>
using namespace std;

int ObtenerYardas(int pulgadas) {
    int yardas = pulgadas / 36;
    if (pulgadas % 36 > 0) {
        yardas++;
    }
    return yardas;
}

int ObtenerExcesoDeTela(int pulgadas) {
    return 36 * ObtenerYardas(pulgadas) - pulgadas;
}

void ReporteCompraDeTela(int pulgadas) {
    cout << "Necesitamos " << pulgadas << " pulgada(s)" << endl;
    cout << "Yardas a comprar: " <<  ObtenerYardas(pulgadas) << endl;
    cout << "Exceso de tela: " << ObtenerExcesoDeTela(pulgadas) << " pulgada(s)" << endl;
    cout << endl;
}

int main(void) {
    ReporteCompraDeTela(71);
    ReporteCompraDeTela(72);
    ReporteCompraDeTela(73);
    return 0;
}

File: tareas/2/24190108/Lista 9/Exceso de tela.py
def obtener_yardas(pulgadas):
    yardas = pulgadas / 36
    if pulgadas % 36 > 0:
        yardas += 1
    return yardas

def obtener_exceso_de_tela(pulgadas):
    return 36 * obtener_yardas(pulgadas) - pulgadas

def reporte_compra_de_tela(pulgadas):
    print(f"Necesitamos {pulgadas} pulgada(s)")
    print(f"Yardas a comprar: {obtener_yardas(pulgadas)}")
    print(f"Exceso de tela: {obtener_exceso_de_tela(pulgadas)} pulgada(s)")
    print()

def main():
    reporte_compra_de_tela(71)
    reporte_compra_de_tela(72)
    reporte_compra_de_tela(73)

main()